controller/response/responder.go

```diff
@@ -55,6 +57,14 @@ func NewRequestResponder(rc *RequestContext) RequestResponder {
 func (rr *RequestResponderImpl) RespondWithApiError(apiError *apierror.ApiError) {
 	log := pfxlog.Logger()
 
+	if logrus.IsLevelEnabled(logrus.DebugLevel) {
+		logger := log
+		if apiError.Cause != nil {
+			logger = logger.WithError(apiError.Cause)
+		}
+		logger.Debugf("returning api error, status: %v, code: %v, msg: %v", apiError.Status, apiError.Code, apiError.Message)
```

Comment:
Is the `if` really needed? Can't we blindly call `logger.Debug` and use `stringz.OrEmpty()`?

Reply:
This if isn't needed. This just seemed like heavier than usual logging logic in what might be a busy code path. I can remove it if you feel strongly about it

Not sure where the stringz.OrEmpty comes in...?

Reply:
mmm mistakenly thought .Cause was in `*string` form here.